third-party-podspecs/GLog.podspec

```diff
@@ -6,12 +6,34 @@ Pod::Spec.new do |spec|
   spec.summary = 'Google logging module'
   spec.authors = 'Google'
 
+  # Keep in sync with ios-configure-glog.sh
```

Comment:
Yeah, I considered that. I guess if people install them through NPM we should have the same file available locally, but cocoapod's linting procedure may not copy these files over. I'll give it a try.